Makefile

```diff
@@ -18,6 +18,10 @@ help:
 	@echo 'Variables'
 	@echo '  ARGS         - Arguments to pass to mut-publish'
 
+publish:
+	rsync -a --delete build/php-library/docs/ source/
```

Comment:
Ehhhhh shoot I should have expected that. How do you folks feel about using git submodules instead of giza's assets system?

For example, you would run `git submodule add -f -b master -- https://github.com/mongodb/mongo-php-library.git build/php-library`, and then `git submodule update` right before the `rsync`